### DISCUSSION TITLE
[Enhancement]  Automatically generate pop-up windows

### DISCUSSION THREAD

**Discussion by Jasonhohn** [xamarin/XamarinCommunityToolkit#1538] [Q&A]
Hello　Excuse me, if I want to load the pop-up window directly on the homepage, how do I do it? 　I tried to do the loading action in the initial function 　 but it didn’t work 　 Thank you

**Comment by pictos** [comment]
Hey @Jasonhohn, can you extend your question a bit more? I feel that I didn't get it
